Cargo.toml

```diff
@@ -4,6 +4,8 @@ version = "0.4.2"
 edition = "2021"
 
 [dependencies]
+log = "0.4.14"
```

Comment:
My current recommendation for logging is [tracing](https://docs.rs/tracing/latest/tracing/). It provides structured logging, async aware spans, and introspection and modularity.

Reply:
If you were to use tracing, you'd replace `systemd-journal-logger` with [tracing-systemd](https://docs.rs/tracing-systemd/latest/tracing_systemd/).

Reply:
+1 on tracing